### PR TITLE
ELPPCKVM-969: [Migration]Add case rhel7_10035_win, update rhel7_10035…

### PR DESCRIPTION
**Pull request by huangyum**
…, rhel7_10044, rhel7_10052

Signed-off-by: Yumei Huang <yuhuang@redhat.com>

### CODE REVIEW

File: virtkvmqe/migration_test_plan/migration/test_scenarios/rhel7_10052.py [hunk 1]
```diff
@@ -50,7 +50,8 @@ def run_case(params):
 
     test.main_step_log('4. On source host, check the migration status')
     check_info = do_migration(remote_qmp=src_remote_qmp,
-                              migrate_port=incoming_port, dst_ip=dst_host_ip)
+                              migrate_port=incoming_port, dst_ip=dst_host_ip,
+                              downtime_val=downtime, speed_val=speed)
```
Comment (wangxianxian) [inline virtkvmqe/migration_test_plan/migration/test_scenarios/rhel7_10052.py]: NameError: global name 'downtime' is not defined
similar to rhel7_10044,  maybe it should be as following:
downtime_val='300', speed_val=speed), with this update, case pass
--> Running case(1/1): RHEL7-10052-[migration] Measure the migration speed(max-bandwidth) |('----transferred_ram_cal:', 1831.136281967163)
('----total_time_cal:', 36.401)
('----speed_cal:', 50.30455981888308)
(2 min 28 sec)--- PASS.

File: virtkvmqe/migration_test_plan/migration/test_scenarios/rhel7_10044.py [hunk 2]
```diff
@@ -58,7 +58,9 @@ def run_case(params):
     test.main_step_log('4. Do live migration.')
     check_info = utils_migration.do_migration(remote_qmp=src_remote_qmp,
                                               migrate_port=incoming_port,
-                                              dst_ip=dst_host_ip)
+                                              dst_ip=dst_host_ip,
+                                              downtime_val=downtime,
+                                              speed_val=speed)
```
Comment (wangxianxian) [inline virtkvmqe/migration_test_plan/migration/test_scenarios/rhel7_10044.py]: NameError: global name 'speed' is not defined,
 there is no assignment for 'speed', so, I think we should use default value 32M/s as following:
speed_val='33554432', with this update, case pass

Comment (huangyum) [reply virtkvmqe/migration_test_plan/migration/test_scenarios/rhel7_10044.py]: Thanks, updated by assigning value for the variant in the beginning.

**Comment by huangyum** [comment]
==>TOTAL : 2
==>PASS : 2 
   1: RHEL7-10044-[migration] Verify migration down time (2 min 56 sec)
   2: RHEL7-10052-[migration] Measure the migration speed(max-bandwidth) (2 min 8 sec)
==>ERROR : 0

**Pull request merged**